backend/api/tests/views/test_views_computetask.py

```diff
@@ -168,17 +184,43 @@ def mock_register_compute_task(orc_request):
         aggregate_task_key = str(uuid.uuid4())
         test_task_key = str(uuid.uuid4())
         predict_task_key = str(uuid.uuid4())
-        done_train_task = self.compute_tasks[ComputeTask.Category.TASK_TRAIN][ComputeTask.Status.STATUS_DONE]
+        datasamples_input = {
+            "identifier": "datasamples",
+            "asset_key": str(self.data_sample.key),
+            "parent_task_key": None,
+            "parent_task_output_identifier": None,
+        }
+        opener_input = {
+            "identifier": "opener",
+            "asset_key": str(self.data_manager.key),
+            "parent_task_key": None,
+            "parent_task_output_identifier": None,
+            # FIXME: "addressable": self.data_manager_data["opener"],
+            "addressable": None,
+            "permissions": self.data_manager_data["permissions"],
+        }
+        model_input = {
+            "identifier": "model",
+            "asset_key": str(self.done_train_task.key),
+            "parent_task_key": None,
+            "parent_task_output_identifier": None,
+            # FIXME: "addressable": self.train_model_data["address"],
+            # FIXME: "permissions": self.train_model_data["permissions"],
+        }
```

Comment:
@acellard I think the current response for CREATE request is buggy: `addressable` and `permissions` are not always as expected. It may be related to the current implementation using "if request", see [here](https://github.com/Substra/substra-backend/blob/main/backend/api/serializers/computetask.py#L353) and [here](https://github.com/Substra/substra-backend/blob/main/backend/api/serializers/computetask.py#L366). But it should be fixed in the next [PR](https://github.com/Substra/substra-backend/pull/458).